dev/vagrant/bootstrap/consul/post/510-poststart-example.yml

```diff
@@ -0,0 +1,3 @@
+- name: "{{ service }} - poststart"
+  debug:
+    msg: This is the first user defined poststart task ( 510-poststart-example.yml )
```

Comment:
Examples in `dev/vagrant/bootstrap/consul/*` can be deleted, right?

Reply:
yepp